test/unit/config_test.rb

```diff
@@ -72,15 +72,15 @@
     describe "when the file is a valid v1.1 file" do
       let(:fixture_name) { "basic" }
       it "should read the file successfully" do
-        expected = %w{create_lockfile reporter type}.sort
+        expected = %w{cli_options create_lockfile credentials reporter type version}.sort
```

Comment:
totally not on you since this already existed... but why are we sorting a static already-sorted array?

Reply:
Ha, yeah, OCD a little.   When reviewing the test and failures it was easier to order them to see what was missing and to make sure it was logical that they should now exist with the changes being made.  I could not find anything that was trying to double validate that the order mattered.